.github/workflows/continuous-integration.yml

```diff
@@ -40,7 +40,10 @@ jobs:
           node-version: ${{ matrix.node }}
       # lint, build, test
       - run: npm install
+      - run: npm run lint
       - run: npm run build
       - run: npm rm tslint
-      - run: npm install ${{ matrix.typescript }}
-      - run: npm run test-cov
+      - run: npm install ${{ matrix.typescript }} --force
+      - run: set +e ; npm run test-cov ; echo $? > tests-exit-code
+      - run: npm install -g coveralls@2 && cat ./coverage/lcov.info | coveralls
```

Comment:
You can use `if: ${{ always() }}` instead: https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#always.

Reply:
Thanks, I was having so much trouble navigating the github actions docs.

Reply:
It's been pretty challenging, but I've been using it in some private monorepos lately so got a bunch of first-hand experience (which seems to be the only way to learn).